modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc
+:_content-type: PROCEDURE
+[id="upi-provisioning-new-hosts-in-a-upi-cluster_{context}"]
+= Provisioning new hosts in a user-provisioned cluster by using the BMO
+
+You can use the Bare Metal Operator (BMO) to provision bare-metal hosts in a user-provisioned cluster by creating a `BareMetalHost` custom resource (CR). 
+
+[NOTE]
+====
+To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
+====
+
+.Prerequisites
+
+* You created a user-provisioned bare-metal cluster.
+* You have baseboard management controller (BMC) access to the hosts.
+* You deployed a provisioning service in the cluster by creating a `Provisioning` CR.
+
+.Procedure
+
+. Create the `Secret` CR and the `BareMetalHost` CR.
+
+.. Save the following YAML in the `bmh.yaml` file:
```

Comment:
Is step `a` part of step 1 or should it be step 2?

Reply:
Part of step 1